Description: Fix ftbfs with GCC-10

Author: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Bug-Debian: https://bugs.debian.org/957319
Forwarded: no

---

--- gtklp-1.3.1.orig/configure.ac
+++ gtklp-1.3.1/configure.ac
@@ -38,7 +38,7 @@ AC_SUBST(LDFLAGS)
 
 # Forte Compiler ############################################################
 FORTE_LDFLAGS=""
-FORTE_CCFLAGS="-Wall"
+FORTE_CCFLAGS="-Wall -fcommon"
 if test x$enable_forte = xyes; then
 	FORTE_LDFLAGS="-lX11"
 	FORTE_CCFLAGS=""
